scripts/installation/install-k8s-1.9.sh

```diff
@@ -0,0 +1,101 @@
+#!/bin/bash
+
+############# Version Information Begin #############
+# kubebernets 1.19 is test ok.
+# istio version: v1.8.0
+# seldon core: v1.4.0
+# harbor: v2.1.0
+# klever 0.1.0
+############# Version Information End ###############
+
+# Set it as k8s master ip.
+export MASTER_IP=$1
+
+# Set harbor NodePort port.
+export HARBOR_PORT=30022
+
+# Set klever-model-registry NodePort port.
+export KLEVER_MODEL_REGISTRY_PORT=30100
+export KLEVER_WEB_PORT=30200
+
+# 
+# Go to manifests directory, it is workdir.
+CWD=$(pwd)
+
+#
+# Install istio, please reference https://istio.io/latest/docs/setup/install/helm/
+#
+curl -L https://istio.io/downloadIstio | ISTIO_VERSION=1.8.0 sh -
+cd istio-1.8.0
+export PATH=$PWD/bin:$PATH
+kubectl create namespace istio-system
+# If you enabled third party tokens, the jwtPolicy option can be deleted
+helm install --namespace istio-system istio-base manifests/charts/base \
+    --set global.jwtPolicy=first-party-jwt
+helm install --namespace istio-system istiod manifests/charts/istio-control/istio-discovery \
+    --set global.hub="docker.io/istio" --set global.tag="1.8.0" \
+    --set global.jwtPolicy=first-party-jwt
+helm install --namespace istio-system istio-ingress manifests/charts/gateways/istio-ingress \
+    --set global.hub="docker.io/istio" --set global.tag="1.8.0" \
+    --set global.jwtPolicy=first-party-jwt
+
+#
+# Install harbor
+#
+kubectl create namespace harbor-system
+helm install harbor harbor --version=v1.4.2 \
+    --repo https://helm.goharbor.io \
+    --set expose.nodePort.ports.http.nodePort=$HARBOR_PORT \
+    --set expose.type=nodePort \
+    --set expose.tls.enabled=false \
+    --set trivy.enabled=false \
+    --set notary.enabled=false \
+    --set persistence.enabled=false \
+    --set trivy.ignoreUnfixed=true \
+    --set trivy.insecure=true \
+    --set externalURL=http://$MASTER_IP:$HARBOR_PORT \
+    --set core.image.tag=v2.1.0 \
+    --set harborAdminPassword="ORMBtest12345" \
+    --set storageInitializer.image=ghcr.io/kleveross/klever-ormb-storage-initializer:v0.0.7 \
+    --set predictiveUnit.defaultEnvSecretRefName=ormb \
+    --namespace harbor-system
```

Comment:
```suggestion
    helm install harbor harbor --version=v1.5.1 \
    --repo https://helm.goharbor.io \
    --set expose.nodePort.ports.http.nodePort=$HARBOR_PORT \
    --set expose.type=nodePort \
    --set expose.tls.enabled=false \
    --set trivy.enabled=false \
    --set notary.enabled=false \
    --set persistence.enabled=false \
    --set trivy.ignoreUnfixed=true \
    --set trivy.insecure=true \
    --set externalURL=http://$MASTER_IP:$HARBOR_PORT \
    --set harborAdminPassword="ORMBtest12345" \
    --namespace harbor-system
```
Test is ok in k8s v1.17.5 and v1.19.2